home-application/src/main/java/com/softserveinc/ita/homeproject/application/api/NewsApiImpl.java

```diff
@@ -29,10 +33,12 @@
 
 @Provider
 @NoArgsConstructor
-public class NewsApiImpl implements NewsApi {
+public class NewsApiImpl extends CommonApiService<NewsDto> implements NewsApi {
 
     private NewsService newsService;
     private HomeMapper mapper;
+    private EntitySpecificationService entitySpecificationService;
```

Comment:
I think we can inject this bean into CommonApi and access it through a protected getter. The same situation for the mapper.